modules/multiwallet.js

```diff
@@ -27,7 +27,7 @@ function init() {
   const particlDataDir = cookie.getParticlDatadirPath();
 
   // watch the folders
-  const glob = join(particlDataDir, 'wallet_*');
+  const glob = join(particlDataDir, 'wallets', 'wallet_*');
```

Comment:
I recall that this didn't work for me, I think I removed this once before.
I tested this with the 0.17 daemon

Reply:
I just updated this to look for wallets in both the default location and the additional 'wallets' dir.